test/e2e/port_test.go

```diff
@@ -127,18 +128,18 @@ var _ = Describe("Podman port", func() {
 		lock2 := GetPortLock("5001")
 		defer lock2.Unlock()
 
-		setup := podmanTest.Podman([]string{"run", "-dt", "-p", "5000:5000", "-p", "5001:5001", ALPINE, "top"})
+		setup := podmanTest.Podman([]string{"run", "--name", "test", "-dt", "-p", "5000:5000", "-p", "5001:5001", ALPINE, "top"})
 		setup.WaitWithDefaultTimeout()
 		Expect(setup.ExitCode()).To(BeZero())
 
 		// Check that the first port was honored
-		result1 := podmanTest.Podman([]string{"port", "-l", "5000"})
+		result1 := podmanTest.Podman([]string{"port", "test", "5000"})
```

Comment:
should this still be `-l`?

Reply:
`-l` doesn't work with `remote`

Reply:
I think it's important to have the latest tested still then.  Can we spin up new tests that skip remote with latest still?

Reply:
Can you clarify whether you mean "it is important to test -l", or "it is important to test -l in this precise exact port test"? There are still a ton of `-l` tests.

Reply:
We should probably have one test for each command on --latest but no more.  Just like we should test every flag.

Reply:
I just checked there are three other tests in port_test.go that test `-l`, I would argue that we don't need this many, but these tests had -l in their test name, so I left them and skipped local.

Reply:
@edsantiago  My main thing is we've coverage.  After hearing our potential low coverage numbers the other day, I hate to remove tests that potentially keep our numbers up a bit.  Not so much these particular tests, but I'm just overall leery about removing potential branches of testing without at least verifying we still have coverage.